### DISCUSSION TITLE
Git: 'configure' is not a git command. See 'git --help'

### DISCUSSION THREAD

**Discussion by chnandu** [community/community#22230] [New to GitHub]
<p>I am new to using Github, I started using git on my new MBP 2019. It has the following version of git installed:<br>
$ git --version<br>
git version 2.24.3 (Apple Git-128)</p>
<p>When I run <code>git configure</code>, it is showing the following error:<br>
$ git configure<br>
git: ‘configure’ is not a git command. See ‘git --help’.</p>
<p>I installed newer version of git using brew and that also didn’t come with configure command:<br>
$ git --version<br>
git version 2.29.2</p>
<p>$ git configure<br>
git: ‘configure’ is not a git command. See ‘git --help’.</p>
<p>What am I missing here ? I am trying to update username &amp; password from terminal for using another GitHub account I own but I am stuck now. I didn’t find any hidden files also to update credentials. Can someone please help ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I assume <code>git config</code> is what you want. :wink:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://git-scm.com/favicon.ico" class="site-icon" width="16" height="16">
      <a href="https://git-scm.com/docs/git-config" target="_blank" rel="noopener">git-scm.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://git-scm.com/docs/git-config" target="_blank" rel="noopener">Git - git-config Documentation</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by chnandu** [comment]
<p>Oops… my bad. I completely missed that.  Thanks for pointing it out.<br>
For some reason my brain read it as “git configure” in all places I looked at online, now I look at them they all say “git config” :slight_smile:<br>
Also another reason I was clueless was due to “git help” not showing “config” as part of help text so I couldn’t catch my mistake.<br>
At any rate, you saved my day otherwise I could still be struggling with it. Thanks.</p>
